main.js

```diff
@@ -134,13 +147,39 @@ function extractData() {
 	};
 }
 
+function getCleanGradesFromExams(exams) {
+	var clean = []
+	var dirty_exams = exams.get()
+	var found = 0;
+	for(var i = 0; i < dirty_exams.length; i++) {
```

Comment:
```suggestion
	for (var i = 0; i < dirty_exams.length; i++) {
```
Also, isn't it possible to use `for (const exam of dirty_exams)` ?